tests/mock_tables/dbconnector.py

```diff
@@ -143,3 +146,10 @@ def keys(self, pattern='*'):
 mockredis.MockRedis.config_set = config_set
 redis.StrictRedis = SwssSyncClient
 SonicV2Connector.connect = connect_SonicV2Connector
+swsssdk.SonicV2Connector = SonicV2Connector
```

Comment:
Why is this required? Isn't SonicV2Connector already swsssdk.SonicV2Connector?

Reply:
Removed

---
In reply to: [518283029](https://github.com/Azure/sonic-snmpagent/pull/162#discussion_r518283029) [](ancestors = 518283029)